### ISSUE TITLE
Enhance Label accessibility with ARIA W3C Recommendation and other approaches

### ISSUE THREAD

**Issue by nikiluk** [OpenEthicsAI/OEL#1]
Use best practices in WCAG2.1 to improve the SVG files of the label and their presentation within the OEL. 

This task requires updated label files to be shared on GitHub as they are not yet there, ask @nikiluk 

```
Feature: Accessible labels

Given the label that is displayed in the web interface, loading from label.openethics.ai
When the labels are accessed by the screen readers
Then their meaning should be clear
And the UI should be not only visible but audible
```

**Comment by iyanuashiri** [comment]
Hello @nikiluk, I have sent a pull request that enhances accessibility of the needed SVGs. Thank you

**Comment by nikiluk** [comment]
@iyanuashiri, I reviewed and merged the pull request! The label is now one step closer to being more accessible for users with impaired vision - Thank you, great job!

**Comment by iyanuashiri** [comment]
Thank you for the opportunuity.
